### DISCUSSION TITLE
Failed to request video translation

### DISCUSSION THREAD

**Discussion by gaptriko** [ilyhalight/voice-over-translation#626] [Q&A] [answered]
перестало работать. около месяца назад. сначала все было хорошо. и в хром и в сафари(хотя там тоже по началу не видело сайты) все работало отлично. сейчас вот такая ситуация. **не удалось запросить перевод видео**

https://github.com/ilyhalight/voice-over-translation/assets/34713832/4418462c-20ba-421f-8c6b-34ce48744d50

**Comment by gaptriko** [comment]
с впн и без него. с разными странами.  cloudflare версия тоже не работает

**Comment by UberMorgott** [comment]
Тоже самое(

**Comment by ilyhalight** [comment, marked as answer]
Обновитесь до последней версии
